apps/opik-backend/src/test/java/com/comet/opik/api/resources/v1/priv/PromptResourceTest.java

```diff
@@ -519,6 +519,93 @@ void retrievePromptVersionsByNameAndCommit__whenApiKeyIsPresent__thenReturnPrope
                 }
             }
         }
+
+        @ParameterizedTest
+        @MethodSource("credentials")
+        @DisplayName("Restore prompt version: when api key is present, then return proper response")
+        void restorePromptVersion__whenApiKeyIsPresent__thenReturnProperResponse(String apiKey, boolean success,
+                io.dropwizard.jersey.errors.ErrorMessage errorMessage) {
+            String workspaceName = UUID.randomUUID().toString();
+
+            mockTargetWorkspace(okApikey, workspaceName, WORKSPACE_ID);
+
+            var prompt = factory.manufacturePojo(Prompt.class).toBuilder()
+                    .id(null)
+                    .createdAt(null)
+                    .lastUpdatedAt(null)
+                    .createdBy(null)
+                    .lastUpdatedBy(null)
+                    .latestVersion(null)
```

Comment:
.lastUpdatedBy(USER)
```suggestion
                    .id(null)
                    .lastUpdatedBy(USER)
                    .createdBy(USER)
                    .template(null)
                    .versionCount(0L)
```
                    .createdBy(USER)